havah/chainscore_basic.go

```diff
@@ -59,10 +63,23 @@ func (s *chainScore) Ex_setRevision(code *common.HexInt) error {
 	if err := scoredb.NewVarDB(as, state.VarRevision).Set(code); err != nil {
 		return err
 	}
-	if err := s.handleRevisionChange(as, int(r), int(code.Int64())); err != nil {
-		return nil
+
+	cid := s.cc.ChainID()
+	s.log.Infof("cid=%#x", cid)
```

Comment:
It would be better to add more description in the log or remove it.

Reply:
The commet above was removed.